changelogs/unreleased/2651-laverya

```diff
@@ -0,0 +1 @@
+bump Kubernetes module dependencies to v0.18.4 to fix https://github.com/vmware-tanzu/velero/issues/2540
```

Comment:
Can you call out in here that it allows code compatibility with v1.18, so readers don't have to jump to the issue?

Reply:
updated